src/Internal/QueryM/ProtocolParameters.purs

```diff
@@ -1,16 +1,23 @@
 -- | A module to get protocol parameters via Ogmios request
 module Ctl.Internal.QueryM.ProtocolParameters
   ( getProtocolParameters
+  , askProtocolParameters
   , module GetProtocolParametersAff
   ) where
 
+import Prelude
+
+import Control.Monad.Reader.Class (asks)
 import Ctl.Internal.QueryM (QueryM, mkOgmiosRequest)
 import Ctl.Internal.QueryM (getProtocolParametersAff) as GetProtocolParametersAff
 import Ctl.Internal.QueryM.Ogmios
   ( ProtocolParameters
   , queryProtocolParametersCall
   )
-import Prelude (unit)
+
+askProtocolParameters :: QueryM ProtocolParameters
```

Comment:
Let's replace `getProtocolParameters` with this implementation. We don't support protocol parameter changes during application runtime anyway (because they can change e.g. during balancing - extremely hard to take into account, but also very rare).